### PR TITLE
Merge Badge component

### PR DESCRIPTION
**Pull request by piyushsh25**
1. there are 4 diff sizes sm,md,lg,xlg.
2. 2 types of badges, one placed at the top right and the other at the top bottom right.
3. they have a default round structure.
Screenshot (183)

### CODE REVIEW

File: css/variable.css [hunk 1]
```diff
@@ -52,6 +52,11 @@
      --size33: 4.125rem;
      --size34: 4.25rem;
      --size35: 4.375rem;
+     --size36: 4.5rem;
```
Comment (FarhanMobashir) [inline css/variable.css:55]: why have so many variables ?

File: components/badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
+@import url("../../css/main.css");
+
+/* avatar  */
+img{
+    border-radius: 50%;
```
Comment (FarhanMobashir) [inline components/badge/badge.css:5]: you can have this as a css variable

**Review by FarhanMobashir — COMMENTED** [review]
looks good to me

**Review by prachi-sahani — COMMENTED** [review]
Looks good to me 👍🏻

File: css/variable.css (hunk 1 above)
Comment (piyushsh25) [reply css/variable.css:55]: Here the size number*2=actual size in px. (which has been converted to rem) i.e it increases by 2px. I wanted to make it consistent enough which ended up creating a lot of variables in sizing.

File: components/badge/badge.css (hunk 2 above)
Comment (piyushsh25) [reply components/badge/badge.css]: done!

**Pull request merged**